### ISSUE TITLE
FPS LAG

### ISSUE THREAD

**Issue by BrazilHax** [haxball/haxball-issues#18]
Players with bad computer can no longer play haxball?

**Comment by matixxx0** [comment]
Why? 
Ping is very good.
U give kick?

**Comment by matixxx0** [comment]
++ i can't join to any rooms

**Comment by BrazilHax** [comment]
Ping is fine, the problem is the same FPS

**Comment by Hongkong-Arfa** [comment]
I have the same problem. Although the ping is low, the fps is unstable, FPS is bouncing. And for some custom map, it is very smooth in flash version, but it is very lag in html5.

**Comment by basro** [comment]
It would be useful to know the computer specs of players having performance issues. (Graphics card, CPU and ram)
Also check if canvas is hardware accelerated in your computer.

**Comment by Logaaan** [comment]
Nah, its not about computer performance ,etc..Its just like some rooms have a bug , sometimes fps just drop and screen freez for all players even for owner of the room ,

**Comment by spammeurfou** [comment]
i play with very bad pc in safe mode, and it's impossible for me too... i have 60 fps and very nice ping but i lag all the time :/

**Comment by slavel** [comment]
I have a good average laptop (HP Pavilion G6 / AMD Radeon, 6 gb), pulls at medium settings PES 2016)))
But in the haxball I feel good control only at 30 fps. 60 fps = rink and inhibition pictures((
